tests/features/notifications.feature

```diff
@@ -0,0 +1,208 @@
+@api @javascript @communities
+Feature: User notifications
+  In order to allow users to stay informed of new content
+  As a site administrator
+  I want to make sure auhtenticated users can subscribe to content on the site
+
+  Background:
+    Given users:
+      | name          | mail                 | pass         | roles              |
+      | administrator | admin@ecample.com    | password123  | administrator      |
+      | authuser      | authuser@example.com | password456  | authenticated user |
+    And these featureSet are enabled
+      | featureSet     |
+      | Notifications  |
+    And 'Article' content:
+      | title       | author        | workbench_moderation_state_new | workbench_moderation_state | language | status |
+      | Article sub | administrator | published                      | published                  | en       | 1      |
+    Given I am not logged in
+    And I go to "/user"
+    And I fill in "Username" with "authuser"
+    And I fill in "Password" with "password456"
+    And I press the "Log in" button

```

Comment:
There is no need for these last 4 steps, you could just use 'And I am logged in as "authuser"'